shared/teams/container.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import Teams from '.'
+import pausableConnect from '../util/pausable-connect'
+import openURL from '../util/open-url'
+
+type DispatchProps = {
+  onCreateTeam: () => void,
+  onJoinTeam: () => void,
+  onReadDoc: () => void,
```

Comment:
Have separate onReadAnnouncement function here, and have it do the same thing as onReadDoc for now (or leave it unlinked), with a TODO to put in the URL for the blog post when it's available